themes/dlts-opensquare/assets/partials/_book-meta.scss

```diff
@@ -76,6 +75,11 @@
   .book-subtitle {
     margin: 0;
     font-size: 1.2rem;
+    background-color: #56068c10;
```

Comment:
Hi, I notice the subtitles now have a background color...  Share your thought process?

Reply:
Hi Laura, that must've been a mistake I might have meant to put it under a different class. Would I have to repush or are you able to confirm which changes that should be added?